tests/Jellyfin.Naming.Tests/Subtitles/SubtitleParserTests.cs

```diff
@@ -1,39 +1,46 @@
-﻿using Emby.Naming.Common;
+﻿using System;
+using Emby.Naming.Common;
 using Emby.Naming.Subtitles;
 using Xunit;
 
 namespace Jellyfin.Naming.Tests.Subtitles
 {
     public class SubtitleParserTests
     {
-        private SubtitleParser GetParser()
-        {
-            var options = new NamingOptions();
-
-            return new SubtitleParser(options);
-        }
-
-        [Fact]
-        public void TestSubtitles()
-        {
-            Test("The Skin I Live In (2011).srt", null, false, false);
-            Test("The Skin I Live In (2011).eng.srt", "eng", false, false);
-            Test("The Skin I Live In (2011).eng.default.srt", "eng", true, false);
-            Test("The Skin I Live In (2011).eng.forced.srt", "eng", false, true);
-            Test("The Skin I Live In (2011).eng.foreign.srt", "eng", false, true);
-            Test("The Skin I Live In (2011).eng.default.foreign.srt", "eng", true, true);
-            Test("The Skin I Live In (2011).default.foreign.eng.srt", "eng", true, true);
-        }
+        private readonly NamingOptions _namingOptions = new NamingOptions();
 
-        private void Test(string input, string language, bool isDefault, bool isForced)
+        [Theory]
+        [InlineData("The Skin I Live In (2011).srt", null, false, false)]
+        [InlineData("The Skin I Live In (2011).eng.srt", "eng", false, false)]
+        [InlineData("The Skin I Live In (2011).eng.default.srt", "eng", true, false)]
+        [InlineData("The Skin I Live In (2011).eng.forced.srt", "eng", false, true)]
+        [InlineData("The Skin I Live In (2011).eng.foreign.srt", "eng", false, true)]
+        [InlineData("The Skin I Live In (2011).eng.default.foreign.srt", "eng", true, true)]
+        [InlineData("The Skin I Live In (2011).default.foreign.eng.srt", "eng", true, true)]
+        public void TestSubtitles(string input, string language, bool isDefault, bool isForced)
         {
-            var parser = GetParser();
+            var parser = new SubtitleParser(_namingOptions);
 
             var result = parser.ParseFile(input);
 
             Assert.Equal(language, result.Language, true);
             Assert.Equal(isDefault, result.IsDefault);
             Assert.Equal(isForced, result.IsForced);
         }
+
+        [Theory]
+        [InlineData("The Skin I Live In (2011).mp4")]
+        public void TestNonSubtitles(string input)
+        {
+            var parser = new SubtitleParser(_namingOptions);
+
+            Assert.Null(parser.ParseFile(input));
+        }
+
+        [Fact]
+        public void TestEmptySubtitlesPath()
```

Comment:
Again, I think these test names could be more verbose to explain exactly what they are testing. It would be nice to decide on a convention to stick to moving forward